application/play/scala/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build app using Maven
+FROM mozilla/sbt:latest
+
+RUN curl -sLJO https://deb.secrethub.io/amd64
+RUN dpkg -i secrethub-cli-amd64.deb
+RUN rm secrethub-cli-amd64.deb
+RUN printf 'demo\ndemo\n' | sbt new playframework/play-scala-seed.g8
+# Install SecretHub
```

Comment:
This comment should be placed a couple lines earlier.

Reply:
Fixed